EmailageJavaScriptWrapper/EmailValidator.js

```diff
@@ -0,0 +1,140 @@
+function emailValidate(requestObject, accountSId, authToken, baseUrl, requestType, dataType, sigMethod, nonce, version) {
+    if (baseUrl == undefined || (baseUrl.length) > 0 == false) {
+        baseUrl = "https://api.emailage.com/EmailAgeValidator/"
+    }
+
+    if (!sigMethod)
+        sigMethod = "HMAC-SHA1";
+
+    if (!version)
+        version = "1.0";
+
+    var authObject = getAuthObject(dataType, accountSId, sigMethod, version);
+
+    if (nonce)
+        authObject.oauth_nonce = nonce;
+
+    var oauthPostModel = getOAuthPostModel(requestType, authObject, baseUrl);
+    var oauthData = getOauthData(oauthPostModel);
+    var sig = oauthSign(authToken + '&', oauthData, sigMethod);
+    var requestUrl = oauthPostModel.action + "?" + getParameterString(oauthPostModel.parameters) + "&oauth_signature=" + percentEncode(sig);
+    var result = "";
+
+    $.ajax({
```

Comment:
jQuery required?

Reply:
Using it just to build the sample form via the requestObj. The core functions are javascript.